internal/mode/static/nginx/runtime/runtime_manager_test.go

```diff
@@ -0,0 +1,107 @@
+package runtime_test
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/go-logr/logr"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime/runtimefakes"
+	"github.com/nginxinc/nginx-plus-go-client/client"
+	ngxclient "github.com/nginxinc/nginx-plus-go-client/client"
+)
+
+var _ = Describe("ManagerImpl UpdateHTTPServers", func() {
```

Comment:
I would include the tests in this file in `manager_test.go` under the Describe block there for the runtime manager.

Reply:
sure that makes sense, will do